.github/dockerfiles/runner-base/Dockerfile

```
@@ -49,3 +49,8 @@ RUN \
   cd $HOME; \
   /bin/sh l_BaseKit*.sh -a --silent --eula accept; \
   rm l_BaseKit*.sh
+
+RUN \
+  wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh; \
+  sh Miniconda3-latest-Linux-x86_64.sh -b
```

WORKDIR is not set, / is not writable by user runner. Also see this hadolint warning https://github.com/hadolint/hadolint/wiki/DL4001 (Either use wget or curl but not both).
```suggestion
RUN \
  cd /tmp; \
  curl -sSLO https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh; \
  sh Miniconda3-latest-Linux-x86_64.sh -b; \
  rm Miniconda3-latest-Linux-x86_64.sh
```